functional/OpenJcePlusTests/build.xml

```diff
@@ -28,6 +28,7 @@
 	<property name="build" location="./bin" />
 	<property name="LIB" value="junit4,hamcrest_core,bcprov_jdk18on"/>
 	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
+	<property environment="env" />
```

Comment:
This branch is [1 commit ahead of](https://github.com/llxia/openjdk-tests/compare/adoptium%3Aaqa-tests%3Amaster...mytest), [1 commit behind](https://github.com/llxia/openjdk-tests/compare/mytest...adoptium%3Aaqa-tests%3Amaster) adoptium/aqa-tests:master.

This file is already in the master branch, not needed in this PR.

Reply:
Apparently it's my workspace ignoring this file, sorry for the confusion.